Atomic/atomic.py

```diff
@@ -326,12 +326,19 @@ def _pull_dockertar_layers(self, repo, imagebranch, temp_dir, input_layers):
     def _pull_image_to_ostree(self, repo, image, upgrade):
         if image.startswith("ostree:"):
             self._check_system_ostree_image(repo, image, upgrade)
-        elif self.args.image.startswith("docker:"):
-            self._pull_docker_image(repo, image.replace("docker:", ""))
+        elif self.args.image.startswith("oci:"):
+            self._check_system_oci_image(repo, image, upgrade)
         elif self.args.image.startswith("dockertar:"):
             self._pull_docker_tar(repo, image.replace("dockertar:", ""))
-        else: # Assume "oci:"
-            self._check_system_oci_image(repo, image, upgrade)
+        else: # Assume "docker:"
+            image = image.replace("docker:", "")
+            try:
+                if ":" in image:
+                    raise Exception, image
+            except:
+                print "Error: you must specify a proper registry to pull from. Proper registries are docker, dockertar, oci, and ostree."
+                sys.exit(1)
```

Comment:
I think this change should not be included.  When "oci:" is specified the image is pulled from a Docker V2 registry through Skopeo. 

"docker:" instead pulls the image from the local Docker (i.e. docker save the image + import the image into OSTree).  This one doesn't seem like the correct default.

What we would like to have though, is the possibility to pull to Docker instead of OSTree.